fhir-openapi/src/main/webapp/META-INF/openapi.json

```diff
@@ -914,9 +914,6 @@
                 ],
                 "summary": "Get the whole system history for Account resources",
                 "operationId": "wholeSystemHistoryAccount",
-                "produces": [
-                    "application/fhir+json"
-                ],
                 "parameters": [
                     {
                         "$ref": "#/components/parameters/_sortParam"
```

Comment:
seems unrelated to your PR, but is this even valid JSON?  don't we need a closing curly-brace here?